### PR TITLE
follow orbit and update gson to 2.8.9

### LINKED ISSUE #588: Maybe updating to gson 2.8.9?
Changelog:

image

It is in general good to keep track of newer versions of upstream dependencies so downstream users won't have to use multiple versions of the same library

### PR DESCRIPTION
**Pull request by cdietrich**
follow orbit and update gson to 2.8.9
Fixes #588

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
Blocked until Orbit 2022-03 M2 is out

File: releng/releng-target/lsp4j.target.target
Comment (jonahgraham) [inline releng/releng-target/lsp4j.target.target:6]: Use `https://download.eclipse.org/tools/orbit/downloads/latest-I/` for now and we can change it later (add it to the checklist on https://github.com/eclipse/lsp4j/issues/541 so it isn't forgotten)

**Review by jonahgraham — APPROVED** [review]
LGTM once it builds.

File: releng/releng-target/lsp4j.target.target
Comment (cdietrich) [reply releng/releng-target/lsp4j.target.target:6]: i would simply wait the 2 weeks for m2

**Pull request merged**
